components/CollaboratorsAmount/SubmitForm.tsx

```diff
@@ -7,23 +8,35 @@ const SubmitForm = () => {
   const { setTab, collaboratorsAmount, setCollaboratorsAmount } =
     useContractBuilderProvider()
 
+  const handleCollaboratorsChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value
+
+    if (value === '' || isNaN(parseInt(e.target.value, 10))) {
+      setCollaboratorsAmount(0)
+
+      return
+    }
+
+    setCollaboratorsAmount(parseInt(e.target.value, 10))
```

Comment:
you are calling the same `parseInt` twice.
- Could you please `DRY` your code?
```suggestion
const intValue = parseInt(e.target.value, 10))
    if (value === '' || isNaN(intValue) {
      setCollaboratorsAmount(0)

      return
    }

    setCollaboratorsAmount(intValue)
```